### PR TITLE
Query multiple tables

### PR DESCRIPTION
**Pull request by elasfarc**
in this PR, I did the following:

- Created owners && species  tables.
- Updated animals tables :
-- Add column species_id which is a foreign key referencing species table
-- Add column owner_id which is a foreign key referencing the owners table
- Write queries (using JOIN)

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -10,4 +10,28 @@ CREATE TABLE animals (
...
+	name VARCHAR(100)
+);
+
+ALTER TABLE animals
+DROP species,
+ADD species_id int,
+ADD CONSTRAINT fk_animal_spe FOREIGN KEY (species_id) REFERENCES species(id),
+ADD owner_id INT,
+ADD FOREIGN KEY (owner_id) REFERENCES owners(id);
+
```
Comment (Emmyn5600) [inline schema.sql:31]: You should modify the animals table:

- Remove column species:heavy_check_mark:
` - Add column species_id which is a foreign key referencing species table`
` -Add column owner_id which is a foreign key referencing the owners table`

File: data.sql [hunk 2]
```diff
@@ -56,3 +56,33 @@ SET weight_kg=weight_kg*-1
...
+
+
+INSERT INTO species(name) VALUES('Pokemon');
+INSERT INTO species(name) VALUES('Digimon');
+
+UPDATE ANIMALS 
+SET species_id = CASE 
+WHEN name like '%mon' THEN 2
+ELSE 1
+END;
```
Comment (Emmyn5600) [inline data.sql]: - [ ] Here. we need to get `id from species WHERE name = 'Digimon'`

File: data.sql [hunk 3]
```diff
@@ -56,3 +56,33 @@ SET weight_kg=weight_kg*-1
...
+
+BEGIN;
+UPDATE animals SET owner_id = 1 WHERE name = 'Agumon';
+UPDATE animals SET owner_id = 2 WHERE name = 'Gabumon' OR name = 'Pikachu';
+UPDATE animals SET owner_id = 3 WHERE name = 'Devimon' OR name = 'Plantmon';
+UPDATE animals SET owner_id = 4 WHERE name = 'Charmander' OR name = 'Squirtle' OR name = 'Blossom';
+UPDATE animals SET owner_id = 5 WHERE name = 'Angemon' OR name = 'Boarmon';
+COMMIT;
+
```
Comment (Emmyn5600) [inline data.sql]: - [ ] After setting owner_id, you need to `SELECT id from owners WHERE full_name`

**Review by Emmyn5600 — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUESTED :recycle:

Hi, @elasfarc 

Great work on implementing the project requirements, however there are some issues to be solved before you get approved but you are almost there:muscle:
   
 - The repo looks professional.:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

Changes required :writing_hand: 

         - Please work on the review too

    Cheers and Happy coding!:clap::clap::clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Comment by AshakaE** [comment]
### Hi @elasfarc  🙋‍♀️, 

# STATUS APPROVED ✔
Thanks for making the requested changes 🙏🏼
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**